doc/sphinx/package_service_mapping.json

```diff
@@ -868,7 +896,121 @@
   "azure-mgmt-appconfiguration": {
     "category": "Management",
     "service_name": "App Configuration",
-    "manually_generated": true
+    "namespaces": [
+      "azure.mgmt.appconfiguration"
+    ]
+  },
+  "azure-mgmt-privatedns": {
+    "category": "Management",
+    "service_name": "Private DNS",
+    "namespaces": [
+      "azure.mgmt.privatedns"
+    ]
+  },
+  "azure-mgmt-frontdoor": {
+    "category": "Management",
+    "service_name": "Network",
+    "namespaces": [
+      "azure.mgmt.frontdoor"
+    ]
+  },
+  "azure-mgmt-hybridcompute": {
+    "category": "Management",
+    "service_name": "Hybrid Compute",
+    "namespaces": [
+      "azure.mgmt.hybridcompute"
+    ]
+  },
+  "azure-mgmt-healthcareapis": {
+    "category": "Management",
+    "service_name": "Healthcare APIs",
+    "namespaces": [
+      "azure.mgmt.healthcareapis"
+    ]
+  },
+  "azure-mgmt-edgegateway": {
+    "category": "Management",
+    "service_name": "Edge Gateway",
+    "namespaces": [
+      "azure.mgmt.edgegateway"
+    ]
+  },
+  "azure-mgmt-deploymentmanager": {
+    "category": "Management",
+    "service_name": "Deployment Manager",
+    "namespaces": [
+      "azure.mgmt.deploymentmanager"
+    ]
+  },
+  "azure-mgmt-peering": {
+    "category": "Management",
+    "service_name": "Peering",
+    "namespaces": [
+      "azure.mgmt.peering"
+    ]
+  },
+  "azure-mgmt-appplatform": {
+    "category": "Management",
+    "service_name": "App Service",
+    "namespaces": [
+      "azure.mgmt.appplatform"
+    ]
+  },
+  "azure-mgmt-botservice": {
+    "category": "Management",
+    "service_name": "Bot Service",
+    "namespaces": [
+      "azure.mgmt.botservice"
+    ]
+  },
+  "azure-mgmt-costmanagement": {
+    "category": "Management",
+    "service_name": "Cost Management",
+    "namespaces": [
+      "azure.mgmt.costmanagement"
+    ]
+  },
+  "azure-mgmt-attestation": {
+    "category": "Management",
+    "service_name": "Attestation",
```

Comment:
I saw the reference here - https://docs.microsoft.com/en-us/azure/iot-dps/concepts-security#attestation-mechanism Not completely sure what's this supposed to be related to :P

Reply:
I'm going to leave it under Attestation until someone screams.